repos/spack_repo/builtin/packages/precice/package.py

```diff
@@ -125,6 +125,12 @@ class Precice(CMakePackage):
     conflicts("%clang@:3.7")
     conflicts("%intel@:16")
 
+    patch(
```

Comment:
Can you add a brief comment above the `patch` directive explaining why it is needed? It could be be as simple as:
```suggestion
    # Add missing '#include<array>'
    patch(
```

That would be useful for future reference when reading the package.

Reply:
@alalazo Done